### PR TITLE
Track clicks on "view insights" button on artist insights module

### PR DESCRIPTION
**Pull request by lilyfromseattle**
We want to track user clicks on the "Show artist insights" button, so we can better quantify how many users dig deeper on the artist when evaluating a work.

NOTE: My test passes even when I remove the simulated button click. Does this mean the tracking is firing even when the button is not clicked or is it just a test issue? I followed existing patterns for click tracking.

### CODE REVIEW

**Comment by sweir27** [comment]
@lilyfromseattle sounds like something might be up with the test (haven't dug too far yet), but have you verified that this works as expected locally?

If you link your reaction --> force locally and set `SEGMENT_WRITE_KEY` to match the env var from staging's configuration, you can watch the tracking calls either in your network inspector or segment's debugger.

If you haven't done this before, I'm happy to walk you through in person!

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 2b764bb4c7dc4427a069940b7e7a6dcb61007447

https://deploy-preview-2208--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/__tests__/ArtistInfo.test.tsx [hunk 1]
```diff
@@ -39,46 +40,64 @@ describe("ArtistInfo", () => {
...
-    wrapper.find("Button").simulate("click")
-    expect(wrapper.find("SelectedExhibitions").html()).toBe(null)
+  describe("Analytics", () => {
+    beforeAll(async () => {
+      wrapper = await getWrapper()
+    })
+    it("tracks click on 'Show artist insights' button", () => {
+      const button = wrapper.find("Button")
+      button.simulate("click")
+      expect(track).toBeCalledWith({
```
Comment (ds300) [inline src/Apps/Artwork/Components/__tests__/ArtistInfo.test.tsx]: Ah, there might be a gotcha here 😅 This track api is super confusing.

`track` itself just returns a decorator function which gets applied when the file `ArtisInfo.tsx` is loaded. So this assertion is only checking that the most recent attempt to annotate a method with `track` was the one for the 'show artist insights' button. It's not checking that a tracking event was dispatched.

To do that, we have this special utility called `mockTracking`. Here's an example usage:

https://github.com/artsy/reaction/blob/58c4a5d5d053dc5ca74f42359cf1668949a3307d/src/Components/Publishing/Byline/__tests__/DateSource.test.tsx#L48-L58

Note that you need to put `jest.unmock('react-tracking')` at the top of the test file for this to work.

File: .vscode/settings.json [hunk 2]
```diff
@@ -47,5 +47,5 @@
     "**/*.snap",
     "**/*.story.tsx"
   ],
-  "debug.node.autoAttach": "on"
+  "debug.node.autoAttach": "off"
```
[the remark this answers is not in the document]
Comment (lilyfromseattle) [reply .vscode/settings.json]: Whoops, that was an accident. Will fix

**Comment by artsyit** [comment]
:rocket: PR was released in v15.11.6 :rocket:

**Pull request merged**